app/src/main/java/fly/speedmeter/grub/MainActivity.java

```diff
@@ -191,7 +193,9 @@ protected void onResume() {
         else
             tempspeed = currentSpeed.getText();
 
-        if (sharedPreferences.getBoolean("isMPH", false))
```

Comment:
why don't you write `isMPH = sharedPreferences.getBoolean("isMPH", false);` only ?